mk/config.mk

```diff
@@ -495,6 +495,15 @@ ifeq ($(CFG_MAP_EXT_DT_SECURE),y)
 $(call force,CFG_DT,y)
 endif
 
+# This option enables OP-TEE to support boot arguments handover via Transfer
+# List defined in Firmware Handoff specification.
+# This feature requires the support of Device Tree.
+CFG_TRANSFER_LIST ?= n
+ifeq ($(CFG_TRANSFER_LIST),y)
+$(call force,CFG_DT,y)
+$(call force,CFG_EXTERNAL_DT,y)
```

Comment:
Transfer list expects the TL is located in secure memory. Should this also enable `CFG_MAP_EXT_DT_SECURE`, for consistency?

Reply:
hmmm, looks reasonable, as `MEM_AREA_TRANSFER_LIST` is mapped to secure. `external_dt.blob` will be pointed to secure when `CFG_TRANSFER_LIST` is enabled. I will add this flag.